Improve API 4.0 documentation (#2812)
wazuh/wazuh-documentation: source/user-manual/api/configuration.rst

Configuration
=============

.. note::

>   Please review the :ref:`Securing API <securing_api>` section for more information on how to protect it.

Reviewer: ```suggestion
  Please review the :ref:`Securing API <securing_api>` section for more information on how to protect the API.
```

.. _api_configuration_file:

Configuration file
------------------

> The API configuration can be found inside ``{WAZUH_PATH}/api/configuration/api.yaml``. All settings are commented out by default. To apply a different configuration, uncomment the desired line.

Reviewer: ```suggestion
The API configuration can be found inside ``{WAZUH_PATH}/api/configuration/api.yaml``. All settings are commented out by default. To apply a different configuration, uncomment and edit the desired line or use the API request to change the configuration (LINK AQUI AL ENDPOINT)
```

.. warning::

>     If running a cluster, the master will NOT send its local API configuration file to the workers. Each node provides its own API, if the configuration file is changed in the master node, it should be updated manually in the workers too. Take care of not overwriting the IP and port in the local configuration of each worker.

Reviewer: ```suggestion
    If running a cluster, the master will NOT send its local API configuration file to the workers. Each node provides its own API, if the configuration file is changed in the master node, the user should manually update the workers API configuration in order to use the same configuration. Take care of not overwriting the IP and port in the local configuration of each worker.
```
